packages/api-plugin-carts/src/xforms/xformCartCheckout.js

```diff
@@ -64,8 +66,8 @@ function xformCartFulfillmentGroup(fulfillmentGroup, cart) {
     selectedFulfillmentOption,
     shippingAddress: fulfillmentGroup.address,
     shopId: fulfillmentGroup.shopId,
-    // For now, this is always shipping. Revisit when adding download, pickup, etc. types
-    type: "shipping"
+    // For now, this is always shipping. Revisit when adding download, pickup, etc. types - done.
```

Comment:
Please remove this no longer relevant comment

Reply:
done